tests/score-board.spec.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const assert = require('assert');
+const ScoreBoardPage = require('../pages/JuiceShop/ScoreBoard');
+
+describe('Hide and show all score board challenge items', () => {
+    before(async () => {
```

Comment:
Please remove white space after `async`, the same is actual for the `it` blocks descriptions